createtransfers/createtransfers.py

```
@@ -0,0 +1,366 @@
+                                                encoding_dir_path,
+                                                encoding)
+                if success is not None:
+                    return success
+        else:
+            success = create_file_and_write(encoding_info['file_name'],
+                                            encoding_dir_path,
+                                            encoding)
+            if success is not None:
+                return success
+
+
+def create_large_test_transfers(**kwargs):
+    """Create large test transfers by copying existing sampledata files
+    multiple times to specific subdirectories of
+    TestTransfers/acceptance-tests/performance/
+    """
+    image_n = 113
+    video_n = 669
+
+    performance_path = os.path.join(
+        HERE, 'TestTransfers', 'acceptance-tests', 'performance')
+    rm_dirs_and_create(performance_path)
+
+    # Create dir images-17M-each-2G-total/ containing 2G of 17M image files.
+    image_path = os.path.join(
+        HERE, 'TestTransfers', 'manualNormalization', 'manualNormalization',
+        'preservation', 'image_8.tif')
+
+    success = create_transfer_collection(performance_path,
+                                         image_path,
+                                         image_n,
+                                         'images-17M-each',
+                                         'tif')
+
+    if success is not None:
+        return ERR_OPERATION_FAILED
+
+    # Create dir video-14M-each-10G-total/ containing 10G of 14M video files.
+    video_path = os.path.join(
+        HERE, 'SampleTransfers', 'Multimedia', 'MakeUp.mov')
+
+    success = create_transfer_collection(performance_path,
+                                         video_path,
+                                         video_n,
+                                         'video-14M-each',
+                                         'mov')
+    if success is not None:
+        ERR_OPERATION_FAILED
```

Shouldn't this return `ERR_OPERATION_FAILED`? That said, given my other comment, the code should be changed to use exceptions instead of returning `success`/`fail` type values.